surfer/viz.py

```diff
@@ -1163,6 +1163,7 @@ def remove_labels(self, labels=None, hemi=None):
                 ll.remove()
 
     def add_morphometry(self, measure, grayscale=False, hemi=None,
+                        colormap=None, min=None, max=None, colorbar=True,
```

Comment:
Looks like you added more options than are documented. Also, maybe these should go after `remove_existing` to ensure backward compat?